wee_slack.py

```diff
@@ -664,6 +664,8 @@ def buffer_input_callback(signal, buffer_ptr, data):
             old = old.replace(r'\/', '/')
             channel.edit_nth_previous_message(msgno, old, new, flags)
     else:
+        if data.startswith('//') or data.startswith(' '):
```

Comment:
`data.startswith(('//', ' '))` (works in Python 2.5+)

Reply:
Ah, nice. Fixed.